src/main/resources/test_files/runtime-spark/DataFrames/ForClauseType1.jq

```diff
@@ -0,0 +1,3 @@
+(:JIQS: ShouldCrash; ErrorCode="XPTY0004"; ErrorMetadata="LINE:2:COLUMN:21:" :)
+for $x as boolean in parallelize(("foo", "bar"))
+return $x
```

Comment:
This file should be named as 'ForClauseTypeCheckError[number].jq'

Reply:
DONE